scripts/AudioManager.js

```diff
@@ -83,16 +83,41 @@ function AudioManager(addEventListener, isTownTune) {
 					audio.volume -= step;
 				} else {
 					clearInterval(fade);
-					audio.pause();
+					playPause(false);
 					audio.volume = oldVolume;
 					if (callback) callback();
 				}
 			}, 100);
 		}
 	}
+	
+	function playPause(play) {
+		if(play) {
+			audio.play();
+		}
+		else {
+			audio.pause();
+		}
+	}
+	function getSrc(game, hour, weather) {
```

Comment:
This function is no longer necessary since the decision is now made by StateManager.